test/gateway/eip20_cogateway/constructor.js

```diff
@@ -0,0 +1,176 @@
+const CoGateway = artifacts.require("EIP20CoGateway");
+MockToken = artifacts.require("MockToken");
+
+const Utils = require("./../../test_lib/utils"),
+    BN = require('bn.js');
+
+const NullAddress = "0x0000000000000000000000000000000000000000";
+
+contract('EIP20CoGateway ', function (accounts) {
+
+
+    describe('constructor', async function () {
```

Comment:
We don't use `describe` at the moment. Instead, it should be `contract('EIP20CoGateway.constructor()'`, at least until we decide a better way.